setup.py

```diff
@@ -5,18 +5,67 @@
 
 from __future__ import absolute_import, print_function
 
+import glob
 import os
 from distutils.sysconfig import get_config_var, get_python_inc
 
 from Cython.Distutils.build_ext import new_build_ext
 from setuptools import setup
 from setuptools.extension import Extension
 
+
+def _find_libucx_libs_and_headers():
```

Comment:
Adding the headers too made this long enough that I decided it was time for it to go into a function.

For context, see https://github.com/rapidsai/ucx-py/pull/1041#discussion_r1596916903

Reply:
Works for me, thanks!